packages/gatsby-theme-api-docs/src/components/type/properties/rows/description.js

```diff
@@ -38,30 +37,46 @@ const DescriptionText = styled.span`
 `;
 
 const Description = ({ property, discriminatorValue }) => {
+  const isConstantLike =
+    property.enumeration && property.enumeration.length === 1;
   const additionalInfos = extractAdditionalInfo(property);
   const renderEnums = property.enumeration && !discriminatorValue;
   return (
     <SpacingsStack scale="s">
-      <DescriptionText>
-        {markdownFragmentToReact(property.description)}
-      </DescriptionText>
-      <SpacingsInline>
-        {renderEnums ? (
-          <Info>
-            {valuesStr}: {property.enumeration.join(', ')}
-          </Info>
-        ) : null}
-        {Object.entries(additionalInfos).map(([info, value], index) => {
-          return (
-            !(typeof value === 'boolean' && !value) && (
-              <Info key={index}>
-                {info}
-                <InfoValue>{value}</InfoValue>
+      {isConstantLike ? (
+        <DescriptionText>
+          <div>
+            <Markdown.InlineCode>{property.enumeration[0]}</Markdown.InlineCode>
+          </div>
+          <div>
+            Only &quot;{property.enumeration[0]}&quot; value must be assigned to{' '}
```

Comment:
These properties can have a normal description text like any others and it must be rendered here like a normal description.  

instead of the div:
```jsx
<DescriptionText>
      {markdownFragmentToReact(property.description)}
</DescriptionText>
```